easybuild/framework/easyconfig/tools.py

```diff
@@ -473,33 +473,41 @@ def find_related_easyconfigs(path, ec):
 
     return sorted(res)
 
-
-def review_pr(pr, colored=True, branch='develop'):
+def review_pr(paths=None, pr=None, colored=True, branch='develop'):
     """
-    Print multi-diff overview between easyconfigs in specified PR and specified branch.
+    Print multi-diff overview between specified easyconfigs or PR and specified branch.
     :param pr: pull request number in easybuild-easyconfigs repo to review
+    :param paths: path tuples (path, generated) of easyconfigs to review
     :param colored: boolean indicating whether a colored multi-diff should be generated
     :param branch: easybuild-easyconfigs branch to compare with
     """
     tmpdir = tempfile.mkdtemp()
 
     download_repo_path = download_repo(branch=branch, path=tmpdir)
     repo_path = os.path.join(download_repo_path, 'easybuild', 'easyconfigs')
-    pr_files = [path for path in fetch_easyconfigs_from_pr(pr) if path.endswith('.eb')]
-
+    
+    if pr:
+        pr_files = [path for path in fetch_easyconfigs_from_pr(pr) if path.endswith('.eb')]
+    elif paths:
+        pr_files = [path[0] for path in paths]
```

Comment:
Hmm, shouldn't we pass the paths without the `generated` part down to `review_pr` instead?

Reply:
in main the paths from `det_easyconfig_paths` are transformed to these tuples and I used those, but I suppose I can call `det_easyconfig_paths` in main again, or store the original paths when it is called the first time